plugin/pkg/admission/serviceaccount/admission.go

```diff
@@ -57,6 +57,8 @@ type serviceAccount struct {
 
 	// LimitSecretReferences rejects pods that reference secrets their service accounts do not reference
 	LimitSecretReferences bool
+	// RequireAPIToken determines whether pod creation attempts are rejected if no API token exists for the pod's service account
+	RequireAPIToken bool
```

Comment:
Agree with Q - an HTTP retry error sounds right, but I don't know the
code-level mechanisms we have for that at this point :)

On Tue, Jun 30, 2015 at 11:29 AM, Jordan Liggitt <notifications@github.com>
wrote:

> In plugin/pkg/admission/serviceaccount/admission.go
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/10523#discussion_r33606695>
> :
>
> > @@ -57,6 +57,8 @@ type serviceAccount struct {
> >
> >  	// LimitSecretReferences rejects pods that reference secrets their service accounts do not reference
> >  	LimitSecretReferences bool
> > +	// RequireAPIToken determines whether pod creation attempts are rejected if no API token exists for the pod's service account
> > +	RequireAPIToken bool
>
>  and downstream consumption/customization
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/10523/files#r33606695>
> .
>